graphql-java-datetime/src/main/java/com/zhokhov/graphql/datetime/DateTimeHelper.java

```diff
@@ -32,7 +32,7 @@
  */
 public final class DateTimeHelper {
```

Comment:
By the way, I was thinking about making this class package protected. But that would be a breaking change for the library so it would require a major version increase

Reply:
I think we can do it, maybe release `2.1.0` with these changes, I don't really think anyone is really using this class outside.

Reply:
I guess using this class outside the lib would be a very bad idea.

Reply:
If we choose to introduce the breaking change, I can also remove the unused methods from this class

Reply:
Yeah, let's do it. I'm ok.

Reply:
I made the change. I do think however that the next version should be 3.0.0 because of this breaking change. It's up for you to decide, update accordingly and release whenever you are comfortable.

Reply:
I think 2.1.0 is ok.